README.rst

```diff
@@ -186,19 +186,19 @@ before creating that mapping.
 **They are not where accounts or credentials are stored - these live on home
 servers. Identity Servers are just for mapping 3rd party IDs to matrix IDs.**
 
-This process is very security-sensitive, as there is obvious risk of spam if it
+This process is very security-sensitive, as there is an obvious risk of spam if it
 is too easy to sign up for Matrix accounts or harvest 3PID data. In the longer
```

Comment:
The suggestion "the longer the term" is not correct.

Additionally, longer term is an adjective; this reads better with a noun in its place. I suggest we hyphenate it.

```suggestion
is too easy to sign up for Matrix accounts or harvest 3PID data. In the long-term,
```